frontend/header.py

```diff
@@ -1,10 +1,43 @@
 import streamlit as st
-import streamlit as st
 
-with open("C:\\Users\\yummy\\OneDrive\\Documents\\h4i\\EDGI_Website\\style.css") as f:
+st.set_page_config(page_title="Page Title", layout="wide")
+
+# Connect file to style.css
+with open("css/style.css") as f:
```

Comment:
I don't believe this works actually (at least on my end). this might need to be "../css/style.css"

Reply:
for some reason ../css/style.css doesn't work for me 😭